src/restart_analysis_and_reload_templates.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import signal
+import sys
+
+
+from helperFunctions.program_setup import program_setup
+from scheduler.Analysis import AnalysisScheduler
+
+
+PROGRAM_NAME = 'Restart FACT Analysis Component'
+PROGRAM_DESCRIPTION = 'Restart FACT Analysis Component and reload Analysis Templates'
+
+
+def shutdown(signum, frame):
+    global run
+    logging.info('shutting down FACT Analysis Component')
+    run = False
+
+
+signal.signal(signal.SIGINT, shutdown)
+signal.signal(signal.SIGTERM, shutdown)
+
+
+def main():
+    args, config = program_setup(PROGRAM_NAME, PROGRAM_DESCRIPTION)
+    analysis_service = AnalysisScheduler(config=config)
+    analysis_service.shutdown()
+    logging.info('Restart Analysis component')
+    analysis_service = AnalysisScheduler(config=config)
```

Comment:
Why Start the Scheduler twice?
Also: The CompareScheduler has to also be started, as it also does view sync.